### PR TITLE
Add Sasl Recipe

### PR DESCRIPTION
**Pull request by brittainhard**
Has no license file in the repo, but the fork has an Apache license.

### CODE REVIEW

File: recipes/sasl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{% set name = "sasl" %}
+{% set version = "0.2.1" %}
+{% set md5 = "ca093d9a3d6f20b79b964a5e5add0202" %}
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Please use the `sha256` hash:

https://conda-forge.org/docs/meta.html#populating-the-hash-field

File: recipes/sasl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/8e/2c/45dae93d666aea8492678499e0999269b4e55f1829b1e4de5b8204706ad9/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Please use the canonical url found here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

File: recipes/sasl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  md5: {{ md5 }}
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Please use `sha256`

File: recipes/sasl/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+#  patches:
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Please delete all comments.

File: recipes/sasl/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/toddlipcon/python-sasl
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Is this the correct URL. It looks like it should be:

https://github.com/cloudera/python-sasl

File: recipes/sasl/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  license: Apache
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: `license: Apache-2.0`

File: recipes/sasl/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  summary: 'Cyrus-SASL bindings for Python'
+  license_family: Apache
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: Please include:

```
  license_file: LICENSE.txt
```

File: recipes/sasl/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    # syntax is module:function.  For example
+    #
+    # - sasl = sasl:main
+    #
+    # Would create an entry point called sasl that calls sasl.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: You're going to need to specify the build script:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

File: recipes/sasl/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/sasl/meta.yaml]: It looks like there is a C-extension, so please add `toolchain` as a `build` requirement:

https://github.com/cloudera/python-sasl/blob/master/setup.py#L17-L21

**Comment by pmlandwehr** [comment]
My only comment is that I got eaten by all of the `libsasl` requirements - those were too hard a nut for me to crack. But godspeed! I'm sure that with some time and love it can be worked out.

**Pull request closed without merging**